dio/README.md

```diff
@@ -262,74 +262,103 @@ final response = await dio.request(
 
 ### Request Options
 
-The `Options` class describes the http request information and configuration.
+The `Options` class describes the HTTP request information and configuration.
 Each Dio instance has a base config for all requests made by itself,
 and we can override the base config with `Options` when make a single request.
 The `Options` declaration as follows:
 
 ```dart
-/// Http method.
+/// The HTTP request method.
 String method;
 
-/// Request base url, it can contain sub path, like: https://dart.dev/api/.
-String? baseUrl;
-
-/// Http request headers.
-Map<String, dynamic>? headers;
-
-/// Timeout for opening url.
-Duration? connectTimeout;
+/// Timeout when sending data.
+///
+/// Throws the [DioException] with
+/// [DioExceptionType.sendTimeout] type when timed out.
+///
+/// `null` or `Duration.zero` meanings no timeout limit.
+Duration? sendTimeout;
 
-/// Whenever more than [receiveTimeout] passes between two events from response stream,
-/// [Dio] will throw the [DioException] with [DioExceptionType.RECEIVE_TIMEOUT].
-/// Note: This is not the receiving time limitation.
+/// Timeout when receiving data.
+///
+/// The timeout represents:
+///  - the timeout during connection established and responded.
+///  - the timeout during data transfer of each bytes event,
+///    rather than the overall timing during the receiving.
```

Comment:
I didn't change the behavior if I write the code correctly. It's the same behavior but didn't annotated in the previous doc.

Reply:
And yes, this is not fully available on the Web.

Reply:
Actually all platforms are supported with the behavior.

Reply:
After looking at the old code again, I am still not sure if that is what it was meant to do :)

> Actually all platforms are supported with the behavior.

Thats good.

Just maybe adjust the text.
```
/// The timeout represents:
///  - a timeout between established connection and 
		the first received response bytes.
///  - the timeout between each following received bytes event,
///    rather than the overall timing during the receiving.
```

Reply:
Sure. Updated